src/index.html

```diff
@@ -12,6 +12,115 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="post"></form>
+
+    <fieldset class="fieldset">
+            <legend> Personal information </legend>
+            
+            <label class="form_field">
+                Surname:
+                <input type="text" name="surname" minlength="2" maxlength="25"
+                required autocomplete="off">
+            </label>
+            
+            <label class="form_field">
+                Name:
+                <input type="text" name="name" autocomplete="off" required>
+            </label>
+
+            <label class="form_field">
+                How old are You?
+                <input type="number" name="age" min="1" max="100" value="12" required>              
```

Comment:
same, fix formatting, alsi remove blank spaces
```suggestion
                <input type="number" name="age" min="1" max="100" value="12" required>
```